src/components/backup/AddFirstWalletStep.tsx

```diff
@@ -106,7 +113,14 @@ export const AddFirstWalletStep = ({ userData }: Props) => {
     });
     InteractionManager.runAfterInteractions(goBack);
     InteractionManager.runAfterInteractions(() => {
-      setTimeout(() => navigate(Routes.IMPORT_SEED_PHRASE_FLOW), 50);
+      setTimeout(
+        () =>
+          navigate(Routes.ADD_WALLET_NAVIGATOR, {
+            screen: Routes.IMPORT_SEED_PHRASE_FLOW,
+            params: { type: 'watch' },
```

Comment:
should we add an enum or type for this param?

Reply:
New React Navigation would give us this 🤔